src/pages/kb/user-guide/querying/scheduling-a-query.md

```diff
@@ -6,23 +6,28 @@ title: Scheduling A Query
 slug: scheduling-a-query
 ---
 
-By default, your query won't have any scheduling but it's super easy to
-adjust!
-
-In the bottom left corner, you'll see the schedule area:
+You can use scheduled query executions to keep your public dashboards updated or to power routine Alerts. By default, your queries will not have a schedule. But this is easy to adjust. In the bottom left corner of the query editor you'll see the schedule area:
 
 <img src="/assets/images/docs/gitbook/query-details.png" width="100%">
 
-Clicking ' Never ' will open a picker for "every x" or at a specific time of
-day. Note: The time you set the scheduling of queries is based on the timezone
-of the computer you're using to set it (then it's translated to UTC).
+Clicking `Never` will open a picker with allowed schedule intervals. Note: The time you set the scheduling of queries is based on the timezone of the computer you're using to set it (then it's translated to UTC).
 
 <img src="/assets/images/docs/gitbook/schedule-modal.png" width="80%">
 
-Once a schedule is set, your query will run automatically as instructed to
-run.
+Once a schedule is set, your query will run automatically.
+
+{% callout info %}
+
+We _strongly recommend_ that you configure schedules for any queries tied to [Alerts]({% link _kb/user-guide/alerts/setting-up-an-alert.md %}). Otherwise, you will not receive alert notifications until a query is manually executed in the UI.
+
+{% endcallout %}
+
+## Scheduled Queries with Parameters
+
+When running queries with parameters on a schedule the scheduler will use the default parameter values for the query execution.
+
+## Failed Scheduled Query Reports
 
-#### Scheduled Queries with Parameters
+If one or many scheduled query executions fails for some reason, Redash will send an hourly report to the queries' owners detailing the failures. These emails go out regularly every hour until there are no more failures. If you ignore them, the emails will mute themselves after the hundredth (100th) occurence of each error. Importantly, these failure report emails run on a job that is independent of your query schedules. It may take up to an hour after a failed query execution before you receive an email report.
```

Comment:
It's better to be more vague about the thresholds, as those might change :)

Reply:
Fixed c43fcf0